modules/friendlywelcome.js

```diff
@@ -594,6 +596,11 @@ Twinkle.welcome.templates = {
 
 	// NON-ENGLISH WELCOMES
 
+	"welcomeen": {
+		description: "welcome for users whose first language is not listed here",
+		linkedArticle: false,
+		syntax: "{{subst:welcomeen}}"
```

Comment:
This template should really use Simple English. Starting a sentence with "Efforts" and using complex conjunctive phrases like "in any case" is hardly Simple English.

Reply:
Good point, although not an issue with the specific request :) I'll have a look

Reply:
Should be fixed now, thanks; I had not thought about that. Feel free to improve further. :)